### ISSUE TITLE
Functionality of the Navigation screen is operable through Tab and arrow keys but there is no such indication

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#816]
**Describe the bug**
Functionality of the Navigation screen is operable through Tab and arrow keys but there is no such indication.

**To Reproduce**
1. Go to Navigation screen and click on any page for a site.
2. If we click on wrench icon on any level, and see the radio buttons.
3. We will notice that these are accessible using Tab and Arrow key but there is no indication.

**Expected behavior**
There should be indication for accessing through different Keyboard keys.

**Additional context**
Navigate all the options available for similar issue and fix.
